developers/blobstream.md

```diff
@@ -97,9 +97,9 @@ along with code for:
 - [The Blobstream X circuits](https://alpha.succinct.xyz/celestia/blobstreamx)
 - [The Blobstream X contract Golang bindings](https://github.com/succinctlabs/blobstreamx/blob/main/bindings/BlobstreamX.go)
 
-Canonical deployments of Blobstream X will be maintained on the
+The first deployments of Blobstream X will be maintained on the
 following chains: Arbitrum One, Base and Ethereum Mainnet. Every 1
-hour, Succinct will post an update to the Blobstream X contract
+hour, the prover/relayer will post an update to the Blobstream X contract
```

Comment:
Isn't the Ethereum one every 4 hours/

Reply:
Thats in the next line, no? These are for the chains aside from Ethereum